content/en/blog/install-ubuntu-utm-apple-silicon.md

```
@@ -243,24 +251,37 @@ alt: UTM Configuration for QEMU display settings
 
-The desktop now uses native macOS resolution for clarity.
+The desktop should now look crisp and properly scaled.
+
+#### Software Updates
 
 Finally, update Ubuntu's packages. Open the menu, select **Software Updater**, and install available updates to ensure optimal performance.
 
-<!-- Check for updates / fixes -->
-### Issues (QEMU)
+Alternatively open the **Terminal** and run:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)